zeebe/broker/src/main/java/io/camunda/zeebe/broker/exporter/stream/ExporterDirector.java

```diff
@@ -280,7 +282,7 @@ private void initContainers() throws Exception {
       container.configureExporter();
     }
 
-    eventFilter = createEventFilter(containers);
+    eventFilter = createEventFilter(containers).and(positionsToSkipFilter);
```

Comment:
:wrench: I'd flip the order to first use the position skip filter. That way the other filters don't even have to decode (possibly corrupted) events.